src/commands/workspace/list.ts

```diff
@@ -10,54 +10,55 @@
 
 import { Command, flags } from '@oclif/command'
 import { cli } from 'cli-ux'
-import * as Listrq from 'listr'
 
 import { CheHelper } from '../../api/che'
-import { accessToken, cheNamespace, skipKubeHealthzCheck } from '../../common-flags'
-import { CheTasks } from '../../tasks/che'
-import { ApiTasks } from '../../tasks/platforms/api'
+import { CheApiClient } from '../../api/che-api-client'
+import { KubeHelper } from '../../api/kube'
+import { accessToken, ACCESS_TOKEN_KEY, cheApiUrl, cheNamespace, CHE_API_URL_KEY } from '../../common-flags'
 
 export default class List extends Command {
   static description = 'list workspaces'
 
   static flags = {
     help: flags.help({ char: 'h' }),
+    quiet: flags.boolean({
```

Comment:
Why do we need this?
Pls remove. User always can use bash script to extract workspaceId from the output.

Reply:
I think it is usable. For end user it will be much easier to parse a list of ID's instead of a table of values.

Reply:
Pls remove it.

Reply:
Okay =(